settings.php

```diff
@@ -97,7 +97,6 @@
     }
     $warning = !method_exists('file_system', 'supports_xsendfile');
     $warningtext = $warning ? $OUTPUT->notification(get_string('settings:presignedurl:coresupport', 'tool_objectfs')) : '';
-    $warningtext .= \tool_objectfs\local\manager::cloudfront_pem_exists();
```

Comment:
Can you still keep this warning in place, in case the pem config item is garbage etc

Reply:
the alternative would be a custom admin setting for the pem with it's own validation. I'll make that as a new issue

Reply:
https://github.com/catalyst/moodle-tool_objectfs/issues/254